specs/decimal/decimal.md

```diff
@@ -45,6 +57,13 @@ data structure may be re-expressed in each language using language-specific cons
 defines the `DecimalValue` structure as the Win32 `DECIMAL` definition syntax is incompatible with
```

Comment:
I'd still suggest renaming `DecimalValue` to just `Decimal` now that the name is not used by the runtime class anymore.